cli/volume/ls.go

```diff
@@ -45,6 +34,21 @@ type ls struct {
 	back bool
 }
 
+type keyValue []vim.KeyValue
+
+func (e *keyValue) String() string {
```

Comment:
This function receiver doesn't need to be by address, and changing it could prevent a possible NPE.